homeassistant/helpers/template.py

```diff
@@ -1361,10 +1361,12 @@ def distance(hass, *args):
     )
 
 
-def is_state(hass: HomeAssistant, entity_id: str, state: State) -> bool:
+def is_state(hass: HomeAssistant, entity_id: str, state: State | list[State]) -> bool:
     """Test if a state is a specific value."""
     state_obj = _get_state(hass, entity_id)
-    return state_obj is not None and state_obj.state == state
+    return state_obj is not None and (
+        isinstance(state, list) and state_obj.state in state or state_obj.state == state
```

Comment:
It would be a little more efficient to turn this around?

```suggestion
        state_obj.state == state or (isinstance(state, list) and state_obj.state in state)
```

As in, this might save a few `isinstance` calls, which are relatively inefficient.